corehq/apps/userreports/examples/examples.md

```diff
@@ -481,6 +481,77 @@ This will lookup the property `age` and substituite its value in the `statement`
 ```
 This will return value of `weight_at_1_year - weight_at_birth`
 
+## Getting forms submitted for a case
+
+```json
+{
+    "type": "get_case_forms",
+    "case_id_expression": {
+        "type": "property_name",
+        "property_name": "case_id"
+    },
+}
+```
+
+## Filter, Map, Reduce, Flatten and Sort expressions
+
+### Getting number of forms of a particular type
+
+```json
+{
+    "type": "reduce_items",
+    "items_expression": {
+        "type": "filter_items",
+        "items_expression": {
+         "type": "get_case_forms",
+         "case_id_expression": {"type": "property_name", "property_name": "case_id"}
+        },
+        "filter_expression": {
+                   "type": "boolean_expression",
+                   "operator": "eq",
+                   "expression": {"type": "property_name", "property_name": "xmls"},
+                   "property_value": "gmp_xmlns"
+        }
+    },
+    "aggregation_fn": "count"
+}
+```
+It can be read as `reduce(count) <- filter(by gmp_xmlns) <- get_case_forms`
```

Comment:
not sure I follow this syntax. I think it might be easier to read if we stick to python like most of the other examples.